reference/3.0/Microsoft.PowerShell.Utility/Get-Random.md

```diff
@@ -106,130 +132,86 @@ PS> Get-Random -InputObject 1, 2, 3, 5, 8, 13 -Count ([int]::MaxValue)
 13
 ```
 
-The value of the **Count** parameter is the **MaxValue** static property of integers.
-
-To return an entire collection in random order, enter any number that is greater than or equal to
-the number of objects in the collection.
-
 ### Example 8: Get a random non-numeric value
 
 This command returns a random value from a non-numeric collection.
 
-```
-PS> Get-Random -InputObject "red", "yellow", "blue"
-yellow
-```
-
-### Example 9: Get a random process
-
-This command gets a randomly selected process from the collection of processes on the computer.
-
-```
-PS> Get-Process | Get-Random
-Handles  NPM(K)    PM(K)      WS(K) VM(M)   CPU(s)     Id ProcessName
--------  ------    -----      ----- -----   ------     -- -----------
-144           4     2080        488    36     0.48   3164 wmiprvse
+```powershell
+"red", "yellow", "blue" | Get-Random
 ```
 
-### Example 10: Run commands on remote computers in random order
-
-This command runs a command on a series of remote computers in random order.
-
-```powershell
-Get-Content Servers.txt | Get-Random -Count (Get-Content Servers.txt).Count |
-  foreach { Invoke-Command -ComputerName $_ -Command 'Get-Process PowerShell' }
+```Output
+yellow
 ```
 
-### Example 11: Use the SetSeed parameter
+### Example 9: Use the SetSeed parameter
 
 This example shows the effect of using the **SetSeed** parameter.
 
-```
-PS> Get-Random -Maximum 100 -SetSeed 23
+Because **SetSeed** produces non-random behavior, it is typically used only to reproduce results,
+such as when debugging or analyzing a script.
 
+```powershell
 # Commands with the default seed are pseudorandom
+Get-Random -Maximum 100 -SetSeed 23
+Get-Random -Maximum 100
+Get-Random -Maximum 100
+Get-Random -Maximum 100
+```
 
-PS> Get-Random -Maximum 100
+```Output
 59
-PS> Get-Random -Maximum 100
 65
-PS> Get-Random -Maximum 100
 21
+```
 
+```powershell
 # Commands with the same seed are not random
+Get-Random -Maximum 100 -SetSeed 23
+Get-Random -Maximum 100 -SetSeed 23
+Get-Random -Maximum 100 -SetSeed 23
+```
 
-PS> Get-Random -Maximum 100 -SetSeed 23
+```Output
 74
-PS> Get-Random -Maximum 100 -SetSeed 23
 74
-PS> Get-Random -Maximum 100 -SetSeed 23
 74
+```
 
+```powershell
 # SetSeed results in a repeatable series
+Get-Random -Maximum 100 -SetSeed 23
```

Comment:
The results here should be the same as the first example since the same seed value is used.